gitless/core.py

```diff
@@ -516,6 +516,23 @@ def lookup_branch(self, branch_name):
             self.git_remote.name + '/' + branch_name, pygit2.GIT_BRANCH_REMOTE)
     return RemoteBranch(git_branch, self.gl_repo)
 
+  def lookup_branches(self, branch_names):
```

Comment:
this code is very similar to the one for `lookup_branch`. Perhaps you should change `lookup_branch` to call `lookup_branches`. e.g:
```python
def lookup_branch(self, branch_name):
  return self.lookup_branches([branch_name])[0]
````